src/main/java/io/cdap/e2e/utils/ElementHelper.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.e2e.utils;
+
+import org.junit.Assert;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Element helper
+ */
+public class ElementHelper {
+  private static final Logger logger = LoggerFactory.getLogger(ElementHelper.class);
+  private static final JavascriptExecutor js = (JavascriptExecutor) SeleniumDriver.getDriver();
+  private static final Actions actions = new Actions(SeleniumDriver.getDriver());
+
+  public static void scrollToElement(WebElement element) {
+    logger.info("Scrolling to the element: " + element);
+    actions.moveToElement(element).perform();
+  }
+
+  public static void scrollToElementUsingJsExecutor(WebElement element) {
```

Comment:
Please add a javadoc to jsExecutor methods explaning these methods should only be used when necessary since test should reproduce what a user would do with the browser.

Reply:
Added.